modules/ztp-pulling-ocp-images.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-updating-gitops-ztp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-pulling-ocp-images_{context}"]
+= Enabling pulling ISO images for the desired {product-title} version
+
+To pull ISO images for the desired {product-title} version, update the `AgentServiceConfig` custom resource (CR) with references to the desired ISO and RootFS images that are hosted on the mirror registry HTTP server.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+* You have {rh-rhacm} with MultiClusterHub enabled.
+
+* You have enabled the assisted service.
+
+.Procedure
+
+. Update the `AgentServiceConfig` CR by running the following command:
++
+[source,terminal]
+----
+$ oc edit AgentServiceConfig
+----
+
+. Add the following entry to the `items.spec.osImages` field in the CR:
++
+[source,yaml,subs="attributes+"]
+----
+- cpuArchitecture: x86_64
```

Comment:
PTAL at the repo guidelines for required content for YAML examples: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#required-fields

In summary, it is important to provide the `apiVersion` and `kind` fields to provide context for the customer and include the complete hierarchy of fields that require updating. You can use `# ...` to snip unimportant sections of the YAML.